ElementX/Sources/Services/Client/ClientProxy.swift

```
@@ -132,6 +135,16 @@ class ClientProxy: ClientProxyProtocol {
         verificationStateSubject.asCurrentValuePublisher()
     }
     
+    private let timelineMediaVisibilitySubject = CurrentValueSubject<TimelineMediaVisibility, Never>(.never)
+    var timelineMediaVisibilityPublisher: CurrentValuePublisher<TimelineMediaVisibility, Never> {
+        timelineMediaVisibilitySubject.asCurrentValuePublisher()
+    }
+    
+    private let hideInviteAvatarsSubject = CurrentValueSubject<Bool, Never>(true)
```

The only case where this may return the initial value, is if the store wasn't initialised properly, since the first initial value from the SDK will always return either the stored value or the default value which is actually `on` (which means do not hide anything).

Not sure if I should bother too much with this considering that if the store doesn't work is an exceptional edge case... but I defaulted the app to hide everything in these cases.

The same happens for the `timelineMediaVisibility` and also for the NSE implementations.
WDYT? Should we keep this exceptional failure by handling it negatively, or should we handle it with the default values of the MSC? (where everything is displayed)

---

We might be changing the MSC... so that the default matches the `off` case